tests/e2e_tests/fluxcal_pol_e2e.py

```diff
@@ -60,46 +137,108 @@ def test_expected_results_e2e(e2edata_path, e2eoutput_path):
     os.mkdir(fluxcal_outputdir_WP2)
 
     ####### Run the DRP walker for WP1
-    print('Running walker for WP1')
+    logger.info('='*80)
+    logger.info('Running processing pipeline')
+    logger.info('='*80)
+
+    logger.info('Running walker for WP1')
     walker.walk_corgidrp(data_filelist_WP1, '', fluxcal_outputdir_WP1)
     fluxcal_file_WP1 = glob.glob(os.path.join(fluxcal_outputdir_WP1, '*abf_cal*.fits'))[0]
     fluxcal_image_WP1 = data.Image(fluxcal_file_WP1)
 
-    #check that the calibration file is configured correctly
-    assert fluxcal_image_WP1.pri_hdr['NAXIS'] == 0
-    assert fluxcal_image_WP1.ext_hdr['DPAMNAME'] == 'POL0'
-    assert fluxcal_image_WP1.data.shape == (1,)
-    assert fluxcal_image_WP1.err.shape == (1,)
-    assert fluxcal_image_WP1.dq.shape == (1,)
+    logger.info('='*80)
+    logger.info('Test Case 2: Output Calibration Product Data Format and Content for WP1')
+    logger.info('='*80)
+    ## check that the calibration file is configured correctly
+    # check HDU0 have no data
+    if fluxcal_image_WP1.pri_hdr == 0:
+        logger.info("HDU0: Header only. Expected: header only. PASS.")
+    else:
+        logger.info(f"HDU0: Contains data. Expected: header only. FAIL.")
```

Comment:
I got this logic when I ran it. Do you? If so, we should fix this. I think the problem is that you need to check the size of data in HDU0, not whether the header of HDU0 == 0.